data/seeds/007-service_type_providers.js

```diff
@@ -0,0 +1,30 @@
+const service_type_providers = [
+  {
+    service_type_id: 1,
+    provider_id: 'd49e0c39-010f-481c-862a-df8c193ed7ac',
```

Comment:
Due to new revelations relating to Okta, for provider_id can you switch to one these:

00unr48onuAmU9sxK5d6
00unr8nm2sJkxkcrH5d6
00unr5lzndqVF13jP5d6
00uk9lxaulDYOiB4H5d6
00unr3s3m2zHx70ck5d6

Reply:
instead of the uuids you used. This change is only for places where provider_id is used

Reply:
Edits made and file pushed.